app/common/test/controllers/labRequestController.js

```diff
@@ -1,7 +1,7 @@
 (function () {
```

Comment:
In [labRequestController.js#L199](https://github.com/edrisse/esaude-emr-poc/blob/1d36d5412ff150ab90979ac1f80e8a1e5e3d67ad/app/common/test/controllers/labRequestController.js#L199) we should be using ```then``` as [```success``` is deprecated](https://code.angularjs.org/1.5.10/docs/api/ng/service/$http#deprecation-notice).
Also not sure if comments should be in pt.